frontend/packages/policy-editor/src/components/PolicyCardRules/PolicyRule/PolicyAccessPackages/PolicyAccessPackageAccordion/PolicyAccessPackageAccordion.tsx

```diff
@@ -0,0 +1,138 @@
+import React, { type ReactElement } from 'react';
+import { useTranslation } from 'react-i18next';
+import classes from './PolicyAccessPackageAccordion.module.css';
+import type { AccessPackageResource, PolicyAccessPackage } from '@altinn/policy-editor';
+import { PolicyAccordion } from '../PolicyAccordion';
+import { useResourceAccessPackageServicesQuery } from 'app-shared/hooks/queries/useResourceAccessPackageServicesQuery';
+import { StudioCheckbox, StudioParagraph, StudioSpinner } from '@studio/components';
+
+const selectedLanguage = 'nb';
+
+interface PolicyAccessPackageAccordionProps {
```

Comment:
We have changed the way we create props to use `type` instead of `interface` 😄 
```suggestion
type PolicyAccessPackageAccordionProps = {
```